data.sql

```diff
@@ -17,4 +17,26 @@ insert into animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg)
 insert into animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) values (10,'Blossom','10-13-1998', 3, true, 17)
 insert into animals (id,name,date_of_birth,escape_attempts, neutered, weight_kg) values (11,'Ditto','05-14-2022', 4, true, 22)
 
-/* End Day2 Queries  */
\ No newline at end of file
+/* End Day2 Queries  */
+
+/* Start Day 3 Queries  */
+
+insert into owners (full_name,age) values ('Sam Smith',34)
+insert into owners (full_name,age) values ('Jennifer Orwell',19)
+insert into owners (full_name,age) values ('Bob ',45)
+insert into owners (full_name,age) values ('Dean Winchester',14)
+insert into owners (full_name,age) values ('Melody Pond',77)
```

Comment:
- [ ] It looks like you forgot to add `Jodie Whittaker` with 38 years of age. Please check it out again and add her too among the owners 😉

Reply:
I just forget to add the query to the data.sql. It was already implemented, please.